modal/image.py

```diff
@@ -1924,6 +1924,8 @@ def run_function(
         region: Optional[Union[str, Sequence[str]]] = None,  # Region or regions to run the function on.
         args: Sequence[Any] = (),  # Positional arguments to the function.
         kwargs: dict[str, Any] = {},  # Keyword arguments to the function.
+        *,
```

Comment:
Would be nicer to have this right after `raw_f` but we'll need to do it gradually.

Reply:
Agreed :(